### DISCUSSION TITLE
[vue-instantsearch] `<ais-search-box>` with multi search?

### DISCUSSION THREAD

**Discussion by donni106** [algolia/instantsearch#6035] [Q&A]
Is there a way to define [multi search](https://www.meilisearch.com/docs/reference/api/multi_search) for vue-instantsearch components like [`<ais-search-box>`](https://www.algolia.com/doc/api-reference/widgets/search-box/vue/)? I would like to receive multiple results based on different keywords provided (for the same index).

**Comment by Haroenv** [comment]
You can have a layout like this:

```vue
<ais-instantsearch ...>
  <ais-search-box/>
  <ais-index index-id="1" ...>
    <ais-hits />
  </ais-index>
  <ais-index index-id="2" ...>
    <ais-hits />
  </ais-index>
</ais-instantsearch>
```

You can't mix the different hits though. It could also be interesting to check out autocomplete, which could fit your use case as well if you're making a dropdown.

**Reply by donni106** [reply]
Thanks for the quick response. Autocomplete is not an option in my case. I want to implement an "or" search, where I need to multisearch for different keywords with a merged result set. As I understand, this is possible with meilisearch but not yet with vue-instantsearch, right?

**Reply by Haroenv** [reply]
You can not disjunctively search for keywords, only for facets. Sorry!

**Reply by donni106** [reply]
Would this be something for the roadmap?

**Reply by Haroenv** [reply]
This is in general solved by adding multiple searchableAttributes, specific rules to turn queries into filters, or using NeuralSearch. For now I don't think there's a need to allow for disjunctive queries directly.

**Reply by donni106** [reply]
Ok thanks, I will try to dig deeper in that topics. Can you point me to somewhere where I can read more about the use case with different keywords connected "or" search?

**Reply by Haroenv** [reply]
I don't think we have a particular guide about this, but: 
- https://www.algolia.com/doc/guides/managing-results/must-do/searchable-attributes/
- https://www.algolia.com/doc/guides/managing-results/rules/rules-overview/in-depth/implementing-rules/#automatic-facet-filter
- https://www.algolia.com/doc/guides/getting-started/neuralsearch/
